Tempo: Separate trace to logs and loki search datasource config (#46655)
grafana/grafana: docs/sources/datasources/tempo.md

-- **Enable Node Graph -** Enables the Node Graph visualization.

### Loki Search

This is a configuration for the Loki search query type.

> -- **Data source -** Loki instance used when searching for traces. The Loki instance must have derived fields configured.

Reviewer: ```suggestion
-- **Data source -** The Loki instance in which you want to search traces. You must configure derived fields in the Loki instance.
```

Author: Committing in the UI isn't working right now, so I'll manually add in those changes
